### PR TITLE
upgrades

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/components/Person/tabs/details/I18nForm.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
+import { cloneDeep } from 'lodash';
```
Comment (edoshor) [inline src/components/Person/tabs/details/I18nForm.js:6]: Are you sure this way we're not importing all of lodash ?
I'm concerned with package size. Shouldn't it be something like `lodash/cloneDeep` ?

File: src/components/Collection/tabs/danger/DangerZoneTab.js [hunk 2]
```diff
@@ -82,10 +82,8 @@ class DangerZoneTab extends Component {
   };
 
   render() {
-    const { collection, status } = this.props;
-    const options                = Object.keys(SECURITY_LEVELS)
-      .map(k => SECURITY_LEVELS[k])
-      .filter(x => x.value !== collection.secure);
+    const { status, collection: { secure } } = this.props;
+    const options                            = Object.keys(SECURITY_LEVELS).map(k => SECURITY_LEVELS[k]);
```
[the remark this answers is not in the document]
Comment (davgur) [reply src/components/Collection/tabs/danger/DangerZoneTab.js:86]: new semantic don't give posability to catch onChange event fron item that was selected

File: src/components/Collections/Container.js [hunk 3]
```diff
@@ -22,18 +22,13 @@ class CollectionsContainer extends Component {
...
   };
 
-  componentWillReceiveProps(nextProps) {
-    const { wipOfCreate } = this.props;
-    const nWip            = nextProps.wipOfCreate;
-    const nErr            = nextProps.errOfCreate;
-    if (wipOfCreate && !nWip && !nErr) {
-      this.askForData(this.getPageNo(true));
+  componentDidUpdate(prevProps) {
+    if (prevProps.wipOfCreate && !this.props.wipOfCreate) {
```
[the remark this answers is not in the document]
Comment (davgur) [reply src/components/Collections/Container.js:26]: look like was think that this flow is not important  , return it back

File: .gitignore [hunk 4]
```diff
@@ -25,3 +25,5 @@ yarn-error.log*
 .editorconfig
 *.swp
 .vcode/
+/src/stylesheets/Kmedia.css
+/yarn.lock
```
[the remark this answers is not in the document]
Comment (davgur) [reply .gitignore:29]: ok, this that I was want ask about all ower projects

File: src/components/Person/tabs/details/I18nForm.js (hunk 1 above)
Comment (davgur) [reply src/components/Person/tabs/details/I18nForm.js:6]: you right

File: package.json [hunk 5]
```diff
@@ -9,51 +9,52 @@
...
-    "node-sass": "^4.13.1",
+    "node-sass": "^5.0.0",
     "node-sass-chokidar": "^1.4.0",
     "npm-run-all": "^4.0.2",
-    "react-app-rewire-vendor-splitting": "^1.1.2",
-    "react-app-rewired": "^1.4.0",
-    "react-scripts": "^1.0.7",
-    "react-test-renderer": "^15.6.1",
-    "why-did-you-update": "0.0.8"
+    "react-app-rewired": "^2.1.6",
```
[the remark this answers is not in the document]
Comment (davgur) [reply package.json:25]: yes we need [customize-cra is wrapper for react-app-rewired](https://github.com/arackaf/customize-cra#how-to-install)

**Review by edoshor — APPROVED** [review]

**Pull request merged**